src/assessor_portal.yml

```diff
@@ -17,9 +17,9 @@
             chdir: "{{ assessor_portal_dir }}"
             cmd: "{{ assessor_portal_dir }}/run.py pull"
         - name: Pull images from Assessor Portal docker-compose.yml
-          community.docker.docker_compose:
-            project_src: "{{ assessor_portal_dir }}"
-            pull: yes
+          ansible.builtin.command:
+            chdir: "{{ assessor_portal_dir }}"
+            cmd: docker compose pull
```

Comment:
Is it worth adding a comment in here that as of now, there is no Ansible module that supports `docker compose` commands, though there likely will be one at some point in the future?

Reply:
Please see commit 2e581b1.

Reply:
@dav3r - Can this conversation be resolved?

Reply:
Yes, sorry for not circling back to this.